chain-abci/src/main.rs

```diff
@@ -84,6 +85,21 @@ fn get_enclave_proxy(opts: &AbciOpt) -> MockClient {
     MockClient::new(get_network_id())
 }
 
+pub fn get_tendermint_filename() -> String {
+    match std::env::var("TENDERMINT_HOME") {
+        Ok(path) => format!("{}/config/genesis.json", path).to_owned(),
+        Err(_) => format!("/Users/leejw51/.tendermint/config/genesis.json").to_owned(),
```

Comment:
just return the error, don't default to `leejw51` -- it's not a default username of everyone

Reply:
Maybe instead of error, return `$HOME/.tendermint/config.genesis.json`?

Reply:
oh, i'll fix it.